app/src/main/java/com/mapbox/maps/testapp/examples/GesturesActivity.kt

```diff
@@ -235,16 +250,21 @@ class GesturesActivity : AppCompatActivity() {
 
     if (enabled) {
       focalPointLatLng = Point.fromLngLat(-0.12968, 51.50325)
-      // TODO add marker
-      // marker = mapboxMap.addMarker(MarkerOptions().position(focalPointLatLng))
+      pointAnnotationManager =
+        binding.mapView.annotations.createPointAnnotationManager(binding.mapView).apply {
+          create(
+            PointAnnotationOptions()
+              .withPoint(focalPointLatLng!!)
```

Comment:
why `!!`?

Reply:
Because in declaration it said it could be null: `private var focalPointLatLng: Point? = null`

Reply:
it's set to concrete point couple of rows above, guess we could revisit it. in any case we try to minimize `!!` usage across the project.

Reply:
yes, but probably we could avoid it 👍